cipd_packages/ruby/tools/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Fetches corresponding dart sdk from CIPD for different platforms, builds
+# an executable binary of codesign to `build` folder.
+#
+# This build script will be triggered on Mac code signing machines.
+
+set -e
+set -x
+
+# Get the directory of this script.
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Get versions from config files.
+RUBY_MAJOR_VERSION=$(cat $DIR/../ruby_metadata.txt | cut -d ',' -f 1)
+RUBY_FILE_NAME=$(cat $DIR/../ruby_metadata.txt | cut -d ',' -f 2)
+
+
+# Create the package structure.
+rm -rf $DIR/../build && mkdir -p $DIR/../build
+mkdir -p $DIR/../build/tools
+
+# Copy files to build directory.
+cp $DIR/auto_relink_dylibs.rb $DIR/../build/tools/auto_relink_dylibs.rb
+cp $DIR/../LICENSE $DIR/../build/LICENSE
+
+# Download ruby.
+mkdir -p $DIR/../cleanup
+curl https://cache.ruby-lang.org/pub/ruby/$RUBY_MAJOR_VERSION/$RUBY_FILE_NAME -o $DIR/../cleanup/$RUBY_FILE_NAME
+
+# Install brew dependencies
+brew install --build-from-source gdbm
```

Comment:
so, if we're installing homebrew deps, it's roughly equivalent to cloning these repos from HEAD. I worry this will break often. However, if we don't plan to rebuild this often, it might not matter too much?

Reply:
Yeah, this will probably break frequently but it is an improvement from the previous state.